doc/api/util.md

```diff
@@ -1393,6 +1393,30 @@ added: v10.0.0
 
 Returns `true` if the value is a native `External` value.
 
+```c++
+#include <js_native_api.h>
+#include <string.h>
+napi_value result;
+static napi_value MyNapi(napi_env env, napi_callback_info info) {
+  int *raw = (int*) malloc(1024);
+  NAPI_CALL(env, napi_create_external(env, (void *) raw, NULL, NULL, &result));
```

Comment:
`NAPI_CALL` is only defined in the Node.js test suite, so I don’t think it’s suitable as part of an example here.

Also, we use left-leaning pointers everywhere (i.e. `void*` instead of `void *`).

Reply:
@addaleax , Thanks.
1. I unwrapped `NAPI_CALL` in the example
2. Pointers are made left leaning